jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -197,13 +198,13 @@ public static Configuration fromEnv() {
     Metrics metrics = new Metrics(metricsFactory);
     Reporter reporter = reporterConfig.getReporter(metrics);
     Sampler sampler = samplerConfig.createSampler(serviceName, metrics);
-    Tracer.Builder builder = new Tracer.Builder(serviceName)
+    JaegerTracerBuilder jaegerTracerBuilder = new JaegerTracerBuilder(serviceName)
         .withSampler(sampler)
         .withReporter(reporter)
         .withMetrics(metrics)
```

Comment:
Think this needs to pass `MetricsFactory`, as `Metrics` is now an internal class.

Reply:
Not sure, as part of the reason for doing this PR is to identify which classes should really be internal. I don't believe `Metrics` needs to be public, so now may be the perfect time to change this - as it is a breaking change anyway.